pg_search/src/postgres/scan.rs

```diff
@@ -96,23 +104,18 @@ pub extern "C" fn amgettuple(
 ) -> bool {
     let mut scan: PgBox<pg_sys::IndexScanDescData> = unsafe { PgBox::from_pg(scan) };
     let iter = unsafe {
-        (scan.opaque as *mut std::vec::IntoIter<(Score, DocAddress, TantivyValue, u64)>).as_mut()
+        // SAFETY:  `amrescan()` leaked an instance of `HitsIterator` into the current Postgres MemoryContext
+        //          and set `scan.opaque` to point to it.
+        (scan.opaque as *mut SearchHitIter).as_mut()
     }
     .expect("no scandesc state");
 
     scan.xs_recheck = false;
 
     match iter.next() {
-        Some((_, _, _, ctid)) => {
-            #[cfg(any(
-                feature = "pg12",
-                feature = "pg13",
-                feature = "pg14",
-                feature = "pg15",
-                feature = "pg16"
-            ))]
```

Comment:
This was unnecessary.  only found it b/c I needed to adjust some surrounding lines.